dev/scripts/remove-ts-machines.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

This provides better portability: https://www.baeldung.com/linux/bash-shebang-lines#using-usrbinenv-bash